addpage.php

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang='en' font-family = 'Segoe UI, Helvetica, Verdana'>
```

Comment:
haha sorry when I meant HTML tag i mispoke, i meant HTML selector in CSS my bad!!